src/server/users/models.js

```diff
@@ -0,0 +1,24 @@
+import db from '../database';
+
+export default class UserManager {
+  static modifyUser(userKey, UserObject, cb) {
```

Comment:
userObject
Or you can just use key, value as a variable name.
bcz UserManager is definitely for User entity and this function also deals only User entiry